src/pyhf/infer/__init__.py

```diff
@@ -43,9 +43,11 @@ def hypotest(
         poi_test (Number or Tensor): The value of the parameter of interest (POI)
         data (Number or Tensor): The data considered
         pdf (~pyhf.pdf.Model): The statistical model adhering to the schema ``model.json``
-        init_pars (:obj:`tensor`): The initial parameter values to be used for minimization
-        par_bounds (:obj:`tensor`): The parameter value bounds to be used for minimization
-        fixed_params (:obj:`tensor`): Whether to fix the parameter to the init_pars value during minimization
+        init_pars (:obj:`tensor` of :obj:`float`): The starting values of the model parameters for minimization.
+        par_bounds (:obj:`tensor` of shape Nx2): The extrema of values the model parameters
+            are allowed to reach in the fit.
+        fixed_params (:obj:`tensor` of :obj:`bool`): The flag to set a parameter constant to its starting
+            value during minimization.
```

Comment:
```suggestion
         value during minimization.
```

Not really sure if this is worth changing, but if we're trying to keep the docstings width to around 80-ish charecters then we might want to minimize our whitespace at the front of continued lines. It seems that (at least in this situation) Sphinx needs only 1 space as an offset to understand it is a continued line and not a new line.

Reply:
if we use 4-space tabs everywhere, why not also in docstrings for consistency?

Reply:
That's fine. Though I don't think we universally apply them. Maybe something to look at later.